packages/reports/src/Containers/statement.tsx

```diff
@@ -88,6 +88,11 @@ const getRowAction: TGetRowAction = (row_obj: TSource | TRow) => {
     if (row_obj.id && ['buy', 'sell'].includes(row_obj.action_type)) {
         const contract_type = extractInfoFromShortcode(row_obj.shortcode)?.category?.toUpperCase();
         const unsupportedContractConfig = getUnsupportedContracts()[contract_type as TUnsupportedContractType];
+        const shouldShowForwardStartingNotification =
+            isForwardStarting(row_obj.shortcode, row_obj.purchase_time || row_obj.transaction_time) &&
```

Comment:
Can we pass a single `row_obj` param to `isForwardStarting` and take all necessary properties from it inside `isForwardStarting`? :)

Reply:
Please, refer to answer https://github.com/binary-com/deriv-app/pull/14181#discussion_r1535102290